ddl/backfilling.go

```diff
@@ -601,6 +601,7 @@ func (w *worker) writePhysicalTableRecord(t table.PhysicalTable, bfWorkerType ba
 				backfillWorkers = append(backfillWorkers, idxWorker.backfillWorker)
 				go idxWorker.backfillWorker.run(reorgInfo.d, idxWorker)
 			case typeUpdateColumnWorker:
+				sessCtx.GetSessionVars().StmtCtx.InCreateOrAlterStmt = true
```

Comment:
might we add a brief comment here (cast diff in select & alter)

Reply:
@tangenta Please update the comment according to the address, thanks.